tools/ocean/divandfull.xml

```diff
@@ -0,0 +1,148 @@
+    export JULIA_DEPOT_PATH="\$PWD:\$JULIA_DEPOT_PATH" &&
+    julia
+        '$__tool_directory__/divandfull.jl'
+        '$input_netcdf_identifier'
+        '$longmin'
+        '$longmax'
+        '$latmin'
+        '$latmax'
+        '$startdate'
+        '$enddate'
+        '$varname'
+        '$selmin'
+        '$selmax'
+        '$bathname'
+    ]]></command>
+    <inputs>
+        <param name="input_netcdf_identifier" type="data" format="netcdf" label="Input your netcdf data"/>
+        <param name="bathname" type="data" format="netcdf" label="Input your bathymetry netcdf file" help="for more info see below."/>
+        <param name="longmin" type="float" min="-180" max="180" value="0" label="Longitude minimal"/>
+        <param name="longmax" type="float" min="-180" max="180" value="0" label="Longitude maximal"/>
+        <param name="latmin" type="float" min="-180" max="180" value="0" label="Latitude minimal"/>
+        <param name="latmax" type="float" min="-180" max="180" value="0" label="Latitude maximal"/>
+        <param name="startdate" type="text" value="yyyy-mm-dd" label="Input the starting date">
+            <sanitizer invalid_char="">
+                <valid initial="string.digits">
+                    <add value="-"/>
+                </valid>
+            </sanitizer>
+        </param>
+        <param name="enddate" type="text" value="yyyy-mm-dd" label="Input the ending date">
+            <sanitizer invalid_char="">
+                <valid initial="string.digits">
+                    <add value="-"/>
+                </valid>
+            </sanitizer>
+        </param>
+        <param name="varname" type="text" value="variable" label="Write the name of the variable of the analysis" help="Example: phosphate">
+            <sanitizer invalid_char="">
+                <valid initial="string.letters">
+                    <add value="_"/>
+                </valid>
+            </sanitizer>
+            <validator type="regex">[0-9a-zA-Z_]+</validator>
+        </param>
+        <param name="selmin" type="integer" min="0" max="100" optional="true" value="0" label="Minimum of the salinity"/>
+        <param name="selmax" type="integer" min="0" max="100" optional="true" value="0" label="Maximum of the salinity"/>
+    </inputs>
+    <outputs>
+        <data name="output_netcdf" label="DIVAnd netcdf output" from_work_dir="outputs_netcdf/*.nc" format="netcdf"/>
+        <!--<collection name="outputs" type="list" label="DIVAnd all outputs">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" directory="/outputs/" format="png"/>
+        </collection>-->
+    </outputs>
+    <tests>
+        <test expect_num_outputs="1">
+            <param name="input_netcdf_identifier" value="data_from_Eutrophication_Med_profiles_2022_unrestricted.nc"/>
+            <param name="bathname" location="https://dox.ulg.ac.be/index.php/s/U0pqyXhcQrXjEUX/download"/>
```

Comment:
is that a stable URL? Maybe zenodo?

Reply:
I asked the original author. He says it stable since 6 years and that it should be stable still for a while.